core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,10 +39,35 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
```

Comment:
Would you be happier with the message if it was `Alluxio Error`? My thinking was to hint to the user which component is complaining (Alluxio or Non-Alluxio)